docs/data-sources/cdn_domains.md

```diff
@@ -0,0 +1,110 @@
+---
+subcategory: Content Delivery Network (CDN)
+---
+
+# huaweicloud_cdn_domains
+
+Use this data source to get a list of CDN domains.
+
+## Example Usage
+
+```hcl
+data "huaweicloud_cdn_domains" "test" {
+  type          = "web"
+  domain_status = "online"
+  service_area  = "mainland_china"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `domain_id` - (Optional, String) Specifies the ID of accelerate domain.
+
+* `name` - (Optional, String) Specifies the name of accelerate domain, using fuzzy matching.
+  Length limit is 1-255 characters.
+
+* `type` - (Optional, String) Specifies the business type of accelerating domain.
+
+* `domain_status` - (Optional, String) Specifies the status of accelerate domain.
+
+* `service_area` - (Optional, String) Specifies the area covered by the acceleration service.
+
+* `enterprise_project_id` - (Optional, String) Specifies the enterprise project ID.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The data source ID.
+
+* `domains` - A list of CDN domains.
+
+The `domains` block supports:
+
+* `id` - The ID of CDN domain.
+
+* `name` - The name of accelerate domain.
```

Comment:
Suggest unifying the use of this word globally, accelerate, accelerating or acceleration?